/* Copyright (C) 2025 Interactive Brokers LLC. All rights reserved. This code is subject to the terms
 * and conditions of the IB API Non-Commercial License or the IB API Commercial License, as applicable. */

syntax = "proto3";

package protobuf;

import "Contract.proto";

option java_outer_classname = "TickByTickRequestProto";
option java_package = "com.ib.client.protobuf";
option csharp_namespace = "IBApi.protobuf";

message TickByTickRequest {
    optional int32 reqId = 1;
	optional Contract contract = 2;
	optional string tickType = 3;
    optional int32 numberOfTicks = 4;
	optional bool ignoreSize = 5;
}
